desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -765,6 +772,17 @@ private JComponent getWindowPanelContent() {
         final JXStatusBar statusBar = WidgetFactory.createStatusBar(_statusLabel);
         RightInformationPanel rightInformationPanel = new RightInformationPanel(_glassPane);
 
+        for (Func<AnalysisJobBuilderWindowImpl, JComponent> pluggableComponent : PLUGGABLE_LABELS_BANNER_COMPONENTS) {
+            final JComponent component = pluggableComponent.eval(this);
+            if (component != null) {
+                if (component instanceof JPanel) {
+                    final PlugabblePanel panel = (PlugabblePanel) component;
```

Comment:
Checking for an instance of a JPanel, but casting to a PluggablePanel seems wrong to me.